service/src/test/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleServiceTest.java

```diff
@@ -197,10 +131,6 @@ void shouldMakeCorrectCallForIsAddHearingsInPastEnabled(Boolean toggleState) {
                 (Runnable) () -> service.isCtscReportEnabled(),
                 (Runnable) () -> service.isCtscEnabled("test name"),
                 buildAttributes("report")),
-            Arguments.of(
```

Comment:
Note that we don't want to delete anymore from this method source. The testing is ensuring that the user attributes aren't persisted across requests (an issue we used to have)